test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -102,4 +112,88 @@ describe('IexecAccessors', async () => {
             '0xfc2178d8b8300e657cb9f8b5a4d1957174cf1392e294f3575b82a9cea1da1c4b',
         );
     });
+
+    it('result for', async function () {
```

Comment:
Why not covering as usual?
```
	function resultFor(bytes32 id)
	external view override returns (bytes memory)
	{
		IexecLibCore_v5.Task storage task = m_tasks[id];
		require(task.status == IexecLibCore_v5.TaskStatusEnum.COMPLETED, 'task-pending');
		return task.resultsCallback; // Expansion - result separation
	}
```
- `Should get resultFor`
- `Should not get resultFor when status is not completed`

Reply:
Was the old architecture in the JS file , I can do that way yes

Reply:
Maybe just wait for one additional feedback before updating

Reply:
The diff is not huge I think we can cover both cases.

Minor suggestion to the suggestion:
* `Should get result of task`
* `Should not get result when task is not completed`

Reply:
splitted in two it tests let me know if that's okay for you